lib/findable.rb

```diff
@@ -0,0 +1,7 @@
+module Findable
+  def find_team_by_team_id(id)
+    tracker.team_manager.teams.find do |team|
+      team.team_id == id
+    end.team_name
+  end
+end
```

Comment:
Add a new line at the end of the file here so it doesn't conflict with the findable.rb I made